src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+export const Button = () => {
+  return (
+    <button className="button-submit" type="submit">
+      Submit
+    </button>
+  );
+};
```

Comment:
You don't need a component that just wraps a button tag. You can use the button tag directly instead of the component.